anonymizer/OpenDiffix.Service/AggregationHooks.fs

```diff
@@ -181,7 +192,22 @@ module StarBucket =
           bucket |> mergeAllAggregatorsInto starBucket
       )
 
-    callback starBucket
+    let executionContext = starBucket.ExecutionContext
+
+    // NOTE this is also true when there is no star bucket (no suppression)
```

Comment:
this would get a more verbose treatment and code without a comment, if we introduce the `SuppressedAnonLowCount` field (see PR description todos)

Reply:
By these comments you mean that `low count` will return `true` even without input? That's basically a 0 AID aggregate.

Reply:
If you want to know if the star bucket is completely empty (no buckets added to it), then you can check a few lines above at `bucket |> mergeAllAggregatorsInto starBucket` and switch a boolean flag there.